modular_pariah/modules/contractor/code/datums/contractor_items.dm

```diff
@@ -136,74 +149,44 @@
 	desc = "Request Syndicate Command to distrupt the station's powernet. Disables power across the station for a short duration."
 	item_icon = "bolt"
 	limited = 2
-	cost = 3
+	cost = 2
 
 /datum/contractor_item/blackout/handle_purchase(datum/contractor_hub/hub)
 	. = ..()
 
-	if (.)
-		power_fail(35, 50)
-		priority_announce("Abnormal activity detected in [station_name()]'s powernet. As a precautionary measure, the station's power will be shut off for an indeterminate duration.", "Critical Power Failure", ANNOUNCER_POWEROFF)
-
-// Subtract cost, and spawn if it's an item.
-/datum/contractor_item/proc/handle_purchase(datum/contractor_hub/hub, mob/living/user)
-
-	if (hub.contract_rep >= cost)
-		hub.contract_rep -= cost
-	else
-		return FALSE
-
-	if (limited >= 1)
-		limited -= 1
-	else if (limited == 0)
-		return FALSE
-
-	hub.purchased_items.Add(src)
-
-	user.playsound_local(user, 'sound/machines/uplinkpurchase.ogg', 100)
-
-	if (item && ispath(item))
-		var/atom/item_to_create = new item(get_turf(user))
-
-		if(user.put_in_hands(item_to_create))
-			to_chat(user, span_notice("Your purchase materializes into your hands!"))
-		else
-			to_chat(user, span_notice("Your purchase materializes onto the floor."))
-
-		return item_to_create
-	return TRUE
+	if(!(.))
+		return
+	power_fail(35, 50)
+	priority_announce("Abnormal activity detected in [station_name()]'s powernet. As a precautionary measure, the station's power will be shut off for an indeterminate duration.", "Critical Power Failure", ANNOUNCER_POWEROFF)
```

Comment:
there has to be a proper proc for this, right?

Reply:
There is the grid check event, but that can turn off power for a lot longer than intended.